src/connector/postgres.rs

```diff
@@ -827,10 +915,40 @@ impl Queryable for PostgreSql {
     }
 }
 
+/// Returns true if a Postgres identifier is considered "safe".
+///
+/// In this context, "safe" means that the value of an identifier would be the same quoted and unquoted. In other words, that it does _not_ need to be quoted.
+///
+/// Spec can be found here: https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
+fn is_safe_identifier(ident: &str) -> bool {
```

Comment:
What about identifiers that are reserved words? For example, postgresql (or CDB) would not like if you have a schema called `user` or `if` and use it in a query without quoting...

Reply:
Hmm good point. Is there a convenient list of the reserved keywords we could use? Is it large? Complicated.